src/app/components/organisms/Tabs/Tabs.sc.ts

```diff
@@ -79,6 +80,17 @@ export const TabHeader = styled.button<TabHeaderProps>`
     }
 `;
 
+export const TabHeaderText = styled.div`
+    /* stylelint-disable-next-line value-no-vendor-prefix */
+    display: -webkit-box;
+    max-height: 32px;
+    overflow: hidden;
+    line-height: 16px;
```

Comment:
Why is this here?
This should not be used like this I think.
The text styling should take care of this prop.